diagnostics-app/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "diagnostics-app",
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "watch": "ng build --watch --configuration development",
+    "test": "ng test"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular/animations": "~12.0.2",
```

Comment:
Same. Don't see that it's used in the code. Do we need it?

Reply:
These were added automatically by the CLI and will be required in the future. For eg. @angular/animations is used in PR #8 for the sidebar module. The are used internally by the modules that are installed.

Reply:
Sounds good if this required for this PR, but not PR 8